doc/examples/plot_circular_elliptical_hough_transform.py

```diff
@@ -128,10 +128,10 @@
 
 # Estimated parameters for the ellipse
 best = result[-1]
```

Comment:
@JDWarner it looks like the orientation doesn't need to be rounded. However, I agree with this statement in general: there's a more concise way to do this:

```python
orientation = best[5]
yc, xc, a, b = np.round(best[:5]).astype(int)
```